projects/m2/021-unique-characters/js/index.js

```diff
@@ -0,0 +1,18 @@
+const string1 = "Hello, Word!"
+const string2 = "lllll"
+
+
+function countLetter(s){
+
+    const count = [];
+    for(i=0;i<s.length;i++){
+        if(!count.includes(s[i])){
+            count.push(s[i])
```

Comment:
Your solution is correct. But it may be interesting to try to solve it using a `Set`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set

As a suggestion consider this:
`new Set(s.split(""))).size`